src/modules/reviewrecords.js

```diff
@@ -107,19 +127,21 @@ class ReviewRecords {
             publish = true,
             validate = true
         } = options;
-        const buffer = await this.prepareReviewRecord(reviewRecord, validate);
+        const rr = await this.prepareReviewRecord(reviewRecord, validate);
+        const buffer = protobuf.ReviewRecord.encode(rr);
         const dagNode = await this.chluIpfs.ipfsUtils.createDAGNode(buffer); // don't store to IPFS yet
         const multihash = IPFSUtils.getDAGNodeMultihash(dagNode);
         if (options.expectedMultihash) {
             if (options.expectedMultihash !== multihash) {
                 throw new Error('Expected a different multihash');
             }
         }
-        if (publish) await this.publishReviewRecord(dagNode, previousVersionMultihash, multihash);
+        this.chluIpfs.events.emit('stored ReviewRecord', { multihash, reviewRecord: rr });
```

Comment:
@fazo96 Why are we emitting all these events that we don't have handlers for? Do you expect these to be consumed/handled in the future?

Reply:
I meant the reading events to be used in the Explorer, so that the underlying service node can be easily monitored and we can know what it's doing apart from the logging which is not easily readable by machines.

I emitted this event because since we have reading events it makes sense to also have writing events? We can live without them though

Reply:
From your earlier comment I now understand why you have these events. Good thinking to put these in.